Before: 59c3b6a356b8fec691e6f168836a0fbb4676c607b5dde6b850df5744edb16d18
After: 05239fd9b0b89a279110feaf2ae8b9effc4045939665a1999f75ee8e38db830d
Patch: 385d1b972cdab92b7443c14a30472f5fd1ebd4165daccd2de545d2a5d4cdad48
--- .simplecov	2012-07-22 20:44:49.502249106 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,11 +0,0 @@
-SimpleCov.start do
-  # project_name 'My Awesome Project'
-
-  # add_group 'Models', 'app/model'
-  # add_group 'Plugins', '(app|lib)/plugins'
-
-  # Remove coverage for test code
-  add_filter 'test'
-end
-
-# vim: ft=ruby et ts=2 sw=2 :
--- test/bootstrap/all.rb	2012-07-22 20:44:49.502249106 +1000
+++ test/bootstrap/all.rb	2012-07-22 21:27:41.793688797 +1000
@@ -1,4 +1,2 @@
 # encoding: utf-8
-APP_ROOT = File.expand_path('../../..',__FILE__)
-require 'rubygems'
-require 'raven/test/bootstrap/all'
+APP_DIR = File.expand_path('../../..',__FILE__)
--- test/bootstrap/unit.rb	2012-07-22 20:44:49.502249106 +1000
+++ test/bootstrap/unit.rb	2012-07-22 21:27:41.793688797 +1000
@@ -1,4 +1,3 @@
 # encoding: utf-8
-$coverage_name = 'unit tests'
 require_relative 'all'
-require 'raven/test/bootstrap/unit'
+
